src/Microsoft.Framework.PackageManager/Install/InstallGlobalCommand.cs

```diff
@@ -67,6 +67,20 @@ public FeedOptions FeedOptions
 
             var temporaryProjectFileFullPath = CreateTemporaryProject(FeedOptions.TargetPackagesFolder, packageId, packageVersion);
 
+            var packageFullPath = Path.Combine(
+                _commandsRepository.PackagesRoot.Root,
+                _commandsRepository.PathResolver.GetPackageDirectory(packageId, new SemanticVersion(packageVersion)));
+
+            if (OverwriteCommands)
+            {
+                if (Directory.Exists(packageFullPath))
+                {
+                    WriteVerbose($"Deleting {packageFullPath}");
```

Comment:
I think this should be WriteInformation. It is pretty important to know when things get deleted